instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaHeaders.java

```diff
@@ -17,12 +17,17 @@
 import org.apache.kafka.common.header.Header;
 import org.apache.kafka.common.header.Headers;
 
+import static brave.kafka.clients.KafkaTracing.log;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 final class KafkaHeaders {
```

Comment:
add a test for this class that has the read-only nature that triggered all of this?

Reply:
Will do